lib/subwayside_ui/gtfs/train_crowding_status.ex

```diff
@@ -0,0 +1,66 @@
+defmodule SubwaysideUi.GTFS.TrainCrowdingStatus do
+  @moduledoc """
+  Provides a few helper functions for quickly accessing crowding information on a
+  per-car granularity.
+  """
+
+  def get_gtfs_realtime_feed do
+    GenServer.call(SubwaysideUi.GTFS, :get_gtfs_feed)
+  end
+
+  def get_feed_func do
+    Keyword.fetch!(Application.get_env(:subwayside_ui, __MODULE__), :feed_func)
+  end
+
+  def get_gtfs_crowding_by_train do
+    {feed_mod, feed_func, feed_opts} = get_feed_func()
```

Comment:
> question: the GenServer this is calling out to (`SubwaysideUi.GTFS`) seems very simple and is essentially running an HTTP fetch every second and caching the result. Have you considered just making the request here so it's fetched when the page is loaded, without needing the GenServer at all? I'm not quite sure how LiveView works so maybe it needs something like a GenServer but I feel like `SubwaysideUi.GTFS` may be simple enough that a GenServer is overkill.

My understanding is without the GenServer we would have 1 GTFS-RT feed in memory per second * number of clients. AWS can certainly handle the load (so I think it's fine that it's constantly updating even when nobody is using SubwaysideUI, the downside of a polling cache) but it could cause an OOM if someone opened a ton of tabs and the container's memory was limited.

`LiveView` has a tick per client - which makes sense if you think about how it might be used to build a user-specific dashboard, etc. There may be a way to do this - it would be super cool to have a pre-configured way to render (a chunk of?) the DOM using a shared state by default to all clients. But, that's not how it's set up here at least from my testing.

This also should decouple any latency in loading the data from blocking the template render, but that's a micro optimization.